cli/src/pcluster/validators/ebs_validators.py

```diff
@@ -45,18 +45,19 @@ class EbsVolumeTypeSizeValidator(Validator):
     """
 
     def _validate(self, volume_type: str, volume_size: int):
-        if volume_type in EBS_VOLUME_TYPE_TO_VOLUME_SIZE_BOUNDS:
-            min_size, max_size = EBS_VOLUME_TYPE_TO_VOLUME_SIZE_BOUNDS.get(volume_type)
-            if volume_size > max_size:
-                self._add_failure(
-                    f"The size of {volume_type} volumes can not exceed {max_size} GiB.",
-                    FailureLevel.ERROR,
-                )
-            elif volume_size < min_size:
-                self._add_failure(
-                    f"The size of {volume_type} volumes must be at least {min_size} GiB.",
-                    FailureLevel.ERROR,
-                )
+        if volume_size is not None:
```

Comment:
You change the logic of this EbsVolumeTypeSizeValidator, can you add a test case for volume_size set to None?

Reply:
Done